biapol_taurus/_project_file_transfer.py

```diff
@@ -129,58 +276,27 @@ def remove_file(self, filename, timeout_in_s: float = 20, wait_for_finish: bool
         False if the timeout was reached
 
         """
-        if not filename.startswith(self.target_project_space):
-            filename = self.target_project_space + filename
+        if not str(filename).startswith(str(self.target_project_space_dir)):
+            filename = self.target_project_space_dir / filename
+        else:
+            filename = Path(filename)
 
-        self._run_command([self.dtrm, filename])
+        proc = self.dm.dtrm('-r', str(filename))
```

Comment:
*mortified*

I fixed `dm` and `proc`. However, I cannot fix `dtrm` because that is the filename of the executable. taurus_datamover gets these attributes automatically from the filenames (see the code there), which has the advantage that it automatically extends its capabilities when new commands are added by the ZIH (or by us in the containers).